Cargo.toml

```diff
@@ -18,6 +18,7 @@ members = [
     "cli-output",
     "client",
     "client-test",
+    "connection-cache",
```

Comment:
New module connection-cache introduced which is refactored out tpu-client to make it generic which can be used for both tpu and other workflows